single_tech_samples/synapseanalytics/sample3_integration_testing/tests/integrationtests/tests/test_synapse_pipeline_ingest_melbparkingdata.py

```diff
@@ -0,0 +1,47 @@
+"""Tests for Synapse pipeline: process_covid_results"""
```

Comment:
is this comment accurate? I'm still reviewing the PR, but I haven't come across another reference to COVID data sets or pipeline that would produce COVID results

Reply:
nope - removing